pkg/xdsrelay/aggregation/v1/aggregation.pb.go

```diff
@@ -0,0 +1,994 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Could we move this to `/pkg/api/aggregation/v1`?

Reply:
done.